Tests/Block/Breadcrumb/BreadcrumbTest.php

```diff
@@ -28,9 +28,9 @@ public function testBlockService()
         $blockService = new BreadcrumbGalleryBlockService_Test(
             'context',
             'name',
-            $this->getMock('Symfony\Bundle\FrameworkBundle\Templating\EngineInterface'),
-            $this->getMock('Knp\Menu\Provider\MenuProviderInterface'),
-            $this->getMock('Knp\Menu\FactoryInterface')
+            $this->prophesize('Symfony\Bundle\FrameworkBundle\Templating\EngineInterface')->reveal(),
+            $this->prophesize('Knp\Menu\Provider\MenuProviderInterface')->reveal(),
+            $this->prophesize('Knp\Menu\FactoryInterface')->reveal()
```

Comment:
Trying to move to prophecy, not easy tho, a lot of tests, but will switch slowly :)